pages/typeform-101.vue

```diff
@@ -0,0 +1,412 @@
+              the features, tips, and tricks that make Typeform a top choice for
+              form builders!
+            </p>
+          </div>
+          <div class="d-flex align-items-center justify-content-center mt-5">
+            <nuxt-img
+              src="/typeform-101/typeform-hero.svg"
+              alt="Hero-Image"
+              class="img-fluid hero__image"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <TypeformKeyFeatures :feature-data="typeformKeyFeatures" />
+    <StandOutFeatureSection :featureData="typeformStandOutFeatures" />
+    <PopularUseCase :featureData="typeformPopularUseCases" />
+    <Limitations :featureData="typeformLimitations" />
+    <FormBuilderComparisionTable
+      heading="Comparing Jotform with Other Leading Form Builders"
+    />
+    <WhyChooseFormester />
+    <FAQwithCategories :categories="typeformFaqs" />
+    <CallToActionSection
+      :content="{
+        heading: 'Start Your Formester Journey Today',
+        subheading:
+          'Start with a Free Sign up and experience the power of Formester firsthand',
+      }"
+    />
+  </div>
+</template>
+
+<script>
+import TypeformKeyFeatures from '../components/competitors/KeyFeatures.vue'
+import StandOutFeatureSection from '../components/competitors/StandOutFeatureSection.vue'
+import PopularUseCase from '../components/competitors/PopularUseCase.vue'
+import Limitations from '../components/competitors/Limitations.vue'
+import FormBuilderComparisionTable from '../components/form-builders-comparision-table.vue'
+
+import FAQwithCategories from '../components/FAQwithCategories.vue'
+import WhyChooseFormester from '../components/jotform101/why-choose-formester.vue'
+import CallToActionSection from '../components/CallToActionSection.vue'
+
+import typeformFaqs from '@/faqs/typeform-101-faqs.js'
+import getSiteMeta from '@/utils/getSiteMeta'
+
+export default {
+  components: {
+    TypeformKeyFeatures,
+    StandOutFeatureSection,
+    PopularUseCase,
+    Limitations,
+    WhyChooseFormester,
+    FormBuilderComparisionTable,
+    FAQwithCategories,
+    CallToActionSection,
+  },
+  computed: {
+    meta() {
+      const metaData = {
+        type: 'website',
+        url: 'https://formester.com/typeform-101/',
+        title: 'Typeform 101 | Designing Forms with Visual Brilliance!',
+        description:
+          "Use Typeform to unleash your form-building powers. Formester provides a guide to exploring Typeform's capabilities, tips, and why it excels!",
+        mainImage: 'https://formester.com/typeform-101-hero-section.png', 
+        mainImageAlt: 'screenshot showing the hero section of typeform 101 page', 
+        keywords: [
+          'Typeform',
+          'Typeform101',
+          'Typeform Guide',
+          'Typeform101 Guide',
+          'Designing Forms with Visual Brilliance!',
+          'web forms for customer support',
+          'web forms productivity',
+          'form builder',
```

Comment:
We may create files like `typeform-data.js` and keep all the relevant data regarding it in that file. Then we can import it separately in this file and use it respectively. That way we keep our data in one single place and tomorrow if we need it in another file or component we can simply import it. More over it will help in updating it, in one single place.